main.go

```diff
@@ -501,11 +524,11 @@ func createPullRequestBranch(repo, pr, action string) error {
 		return fmt.Errorf("%v returned error: %s: %s", gitcmd.Args, out, err.Error())
 	}
 
-	// By default, assume the GitLab project is Northern.tech/Mender/<repo>
-	remoteURL := "git@gitlab.com:Northern.tech/Mender/" + repo
-	if v, ok := githubRepoToGitLabProject[repo]; ok {
-		remoteURL = "git@gitlab.com:" + v
+	remoteURL, err := getRemoteUrlGitLab(org, repo)
+	if err != nil {
+		return fmt.Errorf("getRemoteUrlGitLab returned error: %s", err.Error())
```

Comment:
Nit:
`err.Error()` isn't really necessary. The formatter will call `Error`

Reply:
This is everywhere, so I'll rather leave as-is.